src/constant/study-room.ts

```diff
@@ -0,0 +1,7 @@
+export const studyRoomStep = {
+  PLANNING: '계획',
+  STUDYING: '스터디',
+  RETROSPECT: '회고',
+  RESTING: '휴식',
+  COMPLETED: '완료',
+};
```

Comment:
상수의 경우 as const를 챙겨줘도 좋을 것 같아요.
```suggestion
} as const;
```